### ISSUE TITLE
Disabling line-too-long does not work with triple-quoted-strings

### ISSUE THREAD

**Issue by felipe-olmos-orange** [PyCQA/pylint#7733]
### Bug description

When `fail-on` contains `line-too-long` some long triple-quoted-strings are reported by pylint even after a block disable.
```python
"""I should be ok"""
# pylint: disable=line-too-long
LIST = [
    """
    very loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong
    """,
    "short",
    """
    another looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong
    """
]
```

### Configuration

```ini
[MASTER]
[MESSAGE CONTROL]
fail-on=line-too-long
```

### Command used

```shell
pylint code.py
```

### Pylint output

```shell
************* Module code
code.py:9:0: C0301: Line too long (104/100) (line-too-long)

--------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 10.00/10, -10.00)
```

### Expected behavior

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 0.00/10, +10.00)

### Pylint version

```shell
pylint 2.9.6
astroid 2.6.6
Python 3.9.12 (main, Apr  4 2022, 05:22:27) [MSC v.1916 64 bit (AMD64)]
```

### OS / Environment

Windows 10

**Comment by orSolocate** [comment]
I would say that the `# pylint: disable=` should be stronger than the the pylintrc config.
Even in the case where you plug in your rc file in the command itself, e.g.:
`pylint  --rcfile=<file>  code.py`
I think the disable module line should be stronger.

@felipe-olmos-orange  - did you try to have the disable on this specific line? e.g.

```
"""I should be ok"""
LIST = [
    """
    very loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong  # pylint: disable=line-too-long

    """,
    "short",
```

Also, did you try to disable it in the rcfile and enable it in your module?
`# pylint: enable=line-too-long`

If it follows the same logic, the message wouldn't show where it should... (the same bug in reverse).

**Comment by felipe-olmos-orange** [comment]
@orSolocate The problem is that the string `# pylint: ...`  will make part of the triple-quoted-string because they are multi line.

**Comment by Pierre-Sassoulas** [comment]
Closing as I can't reproduce in latest pylint. pylint 2.9.6 is really old, you should upgrade.

```
pylint a.py --enable=all --enable-all-extensions  # with disable
************* Module a
a.py:2:0: I0011: Locally disabling line-too-long (C0301) (locally-disabled)
a.py:9:0: I0020: Suppressed 'line-too-long' (from line 2) (suppressed-message)
a.py:5:0: I0020: Suppressed 'line-too-long' (from line 2) (suppressed-message)
-------------------------------------------------------------------

Your code has been rated at 10.00/10 (previous run: 4.74/10, +5.26)



(venv)  pierre@pylint  ~/pylint   cleanup ±  nano a.py                                       
(venv)  pierre@pylint  ~/pylint   cleanup ±  pylint a.py --enable=all --enable-all-extensions #without disable

************* Module a
a.py:5:0: C0301: Line too long (118/100) (line-too-long)
a.py:9:0: C0301: Line too long (104/100) (line-too-long)
--------------------------------------------------------------------

Your code has been rated at 0.00/10 (previous run: 10.00/10, -10.00)

**Comment by felipe-olmos-orange** [comment]
I confirm that with pylint 2.14.5 the issue dissapears. Thanks!
